code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -295,8 +296,13 @@
 	if(used == TRUE)
 		to_chat(user, "[used_message]")
 		return
-	used = TRUE
-	to_chat(user, "[use_message]")
+	var/choice = alert(user, "[confirmation_message]",, "Yes", "No")
+	if(choice == "Yes")
```

Comment:
Might be better to just have a check if it's No. If so message and return. 
No need for the else

Reply:
You're right